nrt/data/DynaFlow/VoltageMeasurementsUtilities/reference/outputs/curves/curves.csv

```diff
@@ -0,0 +1,16 @@
+time;VMU_average;VMU_max;VMU_min;U1_step_step_value;U2_step_step_value;U3_step_step_value;connected1_event_state1_value;connected2_event_state1_value;connected3_event_state1_value;
+0.000000;5.000000;5.000000;5.000000;0.000000;5.000000;7.000000;-1.000000;1.000000;-1.000000;
+10.000000;5.000000;5.000000;5.000000;0.000000;5.000000;7.000000;-1.000000;1.000000;-1.000000;
+10.000000;15.000000;15.000000;15.000000;0.000000;15.000000;7.000000;-1.000000;1.000000;-1.000000;
+20.000000;7.333333;15.000000;0.000000;0.000000;15.000000;7.000000;-1.000000;1.000000;-1.000000;
```

Comment:
This line is false (the average should still be 15 but I don't know if we can improve this